react/src/components/atoms/buttons/Button/index.js

```diff
@@ -4,7 +4,9 @@ import classNames from 'classnames';
 import './style.css';
 
 const Button = (button) => {
-  const buttonClasses = classNames({
+  // concat a space at the end of custom classes
+  let buttonClasses = button.classes ? `${button.classes.join(' ')} ` : '';
+  buttonClasses += classNames({
     ma__button: true,
     [`ma__button--${button.usage}`]: button.usage,
     [`ma__button--${button.size}`]: button.size,
```

Comment:
couldnt you do this here instead of line 8
```suggestion
    [`ma__button--${button.size}`]: button.size,
    [button.classes.join(' ')]: button.classes,
```